docs/skopeo-delete.1.md

```diff
@@ -6,17 +6,27 @@ skopeo\-delete - Mark the _image-name_ for later deletion by the registry's garb
 ## SYNOPSIS
 **skopeo delete** [*options*] _image-name_
 
-Mark _image-name_ for deletion.  To release the allocated disk space, you must login to the container registry server and execute the container registry garbage collector. E.g.,
+## DESCRIPTION
+
+Mark _image-name_ for deletion.
+The effect of this is registry-specific; many registries don’t this operation, or don’t allow it in some circumstances / configurations.
+
+**WARNING**: If _image-name_ contains a digest, this affects the referenced manifest, and may delete all tags (within the current repository?) pointing to that manifest.
```

Comment:
```suggestion
**WARNING**: If _image-name_ contains a digest, this affects the referenced manifest, and may delete all tags pointing to that manifest.
```
I am quite confused by that.  In case it's important, could you rephrase?

Reply:
After pushing the same image (same digest) to
- repo1:tag1A
- repo1:tag1B
- repo2:tag2A
- repo2:tag2B

the distribution/distribution implementation of deleting repo1@digest is to untag repo1:tag1A and repo1:tag1B, but not repo2:tag2A and repo2:tag2B.

The specification does not describe any of this in detail.

---

Should the text say something like “all tags within the relevant repository that point to the manifest in question” without the parentheses and the question mark? That’s _probably_ the most reasonable behavior, I just don’t have any data about what various registries do.